source/ncengine/graphics2/diligent/resource/PostProcessBufferResource.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include "graphics2/ShaderTypes.h"
+
+#include "ncengine/graphics/PostProcess.h"
+
+#include "Graphics/GraphicsEngine/interface/Buffer.h"
+#include "Graphics/GraphicsEngine/interface/ShaderResourceVariable.h"
+
+namespace nc::graphics
+{
+struct PostProcessState;
+
+// 
+struct PostProcessDataVariable
+{
+    Diligent::IShaderResourceVariable* variable = nullptr;
+    PostProcessPass::type passId = PostProcessPass::None;
+};
+
+// 
+class PostProcessBufferResource
```

Comment:
We should name this `PostProcessPropertyBufferResource` to distinguish it from the resource that binds the render targets to the shader as an SRV, which I will change to be named `PostProcessSinkBufferResource`